index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Toolbar -->
+    <nav class="toolbar">
+        <button type="button"><img src="logo/Icon - Menu.png" alt=""></b>
+    </nav>
+    <!-- Headline -->
+    <section class="headline">
+        <div class="headline_content">
+            <div class="title">
+                <h1>
+                    Hey there. <br> I'm Patience. <br> <span>I'm a software <br> developer</span>
+                </h1>
+            </div>
+            <div class="description">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you
+                    like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+            </div>
+            <div class="socialIcons">
+                <ul>
+                    <li>
+                        <a href="https://twitter.com/CheptooGyts">
+                            <img src="logo/twitter.png" alt="twitter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/cheptoo-patience-73499015a">
+                            <img src="logo/Linkedin icon.png" alt="Linkedin">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="logo/medium.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/Cheptoo21">
+                            <img src="logo/github.png" alt="github">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="logo/angellist.png" alt=""></a>
+                    </li>
+
+                </ul>
+            </div>
+            
+
+        </div>
+    </section>
```

Comment:
[OPTIONAL]
- [ ] Kindly wrap your whole `section` with the `main` tag.

---

- Kindly take a look at the picture which displays the tags that have to be used inside the `body`:
  -  First of all the `header`, which in some cases includes the `nav` inside of it, like our portfolio, and in some cases it doesn't.
  - Secondly, which is not displayed in the picture, the `main` tag, which wraps everything related to your visible content (sections, articles, divs, images, etc...).
  - Last but not least, the `footer`.
  
- All in all, your `body` should consist of 3 tags: `header`, `main`, and `footer`.

    ![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/102612396/ded8352e-ecf2-4782-8def-81e7815ccf6c)